src/widgets/ActivityGroups/ui/ActivityCard/ActivityCardRestartResume.tsx

```diff
@@ -83,15 +83,14 @@ export const ActivityCardRestartResume = ({
               padding: '10px 10px',
               transition: 'all 0.2s',
               minWidth: '120px',
+              mt: 2,
             }}
           >
-            <Box display="flex" flex={1} flexDirection="row" mt={2}>
-              <img src={ActivityRestartIcon} alt="Activity Restart Icon" />
+            <img src={ActivityRestartIcon} alt="Activity Restart Icon" />
```

Comment:
Do we consider i18n in out alt tags?

Reply:
Another great observation! Yes, this should be translated, I'll add it. I've seen i18n being neglected elsewhere in the web app. 😭😭